2024-07-08 09:54:45,974 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2024-07-08 09:54:45,974 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Users\Vishnu\AppData\Local\Programs\Python\Python310\lib\site-packages\instabot\bot\bot.py
2024-07-08 09:54:45,975 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2024-07-08 09:54:46,158 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2024-07-08 09:54:46,158 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2024-07-08 09:54:46,158 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2024-07-08 09:54:46,159 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 5 minutes.
